Dockerfile

```diff
@@ -1,22 +1,22 @@
 FROM python:3.5
-
 USER root
-
 RUN apt-get update && apt-get install -y \
   build-essential \
   libxml2-dev \
   libxslt1-dev \
   python3-dev \
   unzip \
+  libmagickwand-dev \
+  ghostscript \
   zlib1g-dev
-  
-RUN pip install --upgrade pip
 
+RUN pip install --upgrade pip
 COPY requirements.txt ./
 COPY setup ./
 COPY rosie.py ./
 COPY rosie ./rosie
 COPY config.ini.example ./
+COPY config.ini ./
```

Comment:
Done, i removed it. Strange the first time i tried to build the image i got error without it... Any way now its ok